src/components/login/GithubLoginButton.tsx

```diff
@@ -0,0 +1,54 @@
+import { type Provider } from '@supabase/supabase-js'
+import { useTransition } from 'react'
+import { Button } from '../ui/button'
```

Comment:
This is also a really small change, but I'd try using the `@` alias to make imports more flexible. It will help ensure all of the imports are still correct even if we move this file somewhere else!

for example:

```typescript
import { Button } from '@/components/ui/button'
```

You can also change this for the `toast`, and `ButtonLoadingSpinner` imports!